### PR TITLE
minor improvements to Wrangler object

### PR DESCRIPTION
**Pull request by dsuplica**
- ensure that training/testing are being done on an equal number of conditions by re-equalizing after split
    - i.e. if training is [0,0,1,1,1] it is now [0,0,1,1] to avoid bias
    - good idea?
- A couple overrides for wrangler.subs
    - give included_subs to only run on a subset of subject directories
    - give dropped_subs to run on all subject directories except a subset
    - give neither to run on everything
- Remove resampling code
- other minor bug fixes

### CODE REVIEW

**Comment by dsuplica** [comment]
re equalize training set by appending missing labels to test set (to not fully waste)

eg

train: 3A, 2B
test: 1A, 2B

currently goes to 2A2B -> 1A1B, but this would give 2A2B -> 2A2B
